{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/urls.py

```diff
@@ -9,24 +9,4 @@
     url(r'^register/user/', RegistrationView.as_view(), name='user-register'),
```

Comment:
These views don't have anything to do with djoser - so the preceding if probably doesn't make sense